### ISSUE TITLE
Error: src refspec master does not match any

### ISSUE THREAD

**Issue by anton21m** [salesforce/bazoku#8]
```
bazel run go:bazoku-deployment --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64
INFO: Analyzed target //go:bazoku-deployment (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
Target //go:bazoku-deployment up-to-date:
  bazel-bin/go/bazoku-deployment-bazoku
INFO: Elapsed time: 0.424s, Critical Path: 0.00s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action
INFO: Build completed successfully, 1 total action
------------------------------
checking authentication..
heroku-test@gmail.com
------------------------------
Starting heroku deployment of reterterter using bazoku..
Adding heroku-binary-buildpack to reterterter..
------------------------------
error: src refspec master does not match any
error: failed to push some refs to 'https://git.heroku.com/reterterter.git'

```
I get failed error. How to fix it?

**Comment by kriscfoster** [comment]
Hi @anton21m, we'll be taking a look at this sometime this week. Thank you for raising.

**Comment by anton21m** [comment]
I found problems here:
https://github.com/salesforce/bazoku/blob/43874a3ee30e0ce3289382c326d1f4c27a028268/bazoku.bzl#L71
it need replace on `git push heroku main -f --quiet`

**Comment by kriscfoster** [comment]
@anton21m this should be fixed with v0.3.0 which was just released. Thank you for raising the issue & let us know how it goes.
